src/client/App.js

```diff
@@ -1,28 +1,22 @@
 import React, { Suspense } from "react";
 import { Switch, Route } from "react-router-dom";
-import fflip from "fflip";
+import PropTypes from 'prop-types'
 import GuidePage from "./pages/GuidePage";
 import PageNotFound from "./pages/PageNotFound";
 import RedirectToGuide from "./pages/RedirectToGuide";
 import RetroCertsAuthPage from "./pages/RetroCertsAuthPage";
 import routes from "../data/routes";
-import fflipConfig from "../data/fflipConfig";
 
-// Load default feature flag values.
-fflip.config(fflipConfig);
-
-export default function App() {
-  if (process.env.NODE_ENV === "development"
-      || String(process.env.ENABLE_RETRO_CERTS) === "1") {
-    fflip.features.retroCerts.enabled = true;
-  }
+export default function App(props) {
+  const hostname = props.hostname || window.location.hostname
+  const retroCertsEnabled = hostname !== "unemployment.edd.ca.gov";
```

Comment:
minor, maybe rename retroCertsEnabled to isProduction for future readers? since it gets assigned to RetroCertsPage below

Reply:
Good idea, done.